pom.xml

```diff
@@ -11,7 +11,7 @@
 
   <groupId>org.bdgenomics.bdg-formats</groupId>
   <artifactId>bdg-formats</artifactId>
-  <version>0.7.1-SNAPSHOT</version>
+  <version>0.7.2-SNAPSHOT</version>
```

Comment:
ah, I see this was to have [https://github.com/bigdatagenomics/adam/pull/988] depend on a unique version it could use.

might be nice to have a test maven server we could upload builds to using their git SHA as their maven version, as is discussed somewhat [here](https://github.com/bigdatagenomics/adam/issues/713). out of scope here though, so nevermind!